src/chains/arbitrum/nodes.ts

```diff
@@ -0,0 +1,6 @@
+import { consensusNodes as mainnetConsensusNodes } from '@/chains/mainnet/nodes/consensus';
+import { executionNodes as mainnetExecutionNodes } from '@/chains/mainnet/nodes/execution';
+import { Node } from '@/types';
+
+export const executionNodes: Node[] = mainnetExecutionNodes;
+export const consensusNodes: Node[] = mainnetConsensusNodes;
```

Comment:
Similar for arbitrum, no consensus, and the only execution node is their own Nitro node: https://github.com/OffchainLabs/nitro